pythonforandroid/distribution.py

```diff
@@ -236,3 +266,23 @@ def pretty_log_dists(dists, log_func=info):
 
     for line in infos:
         log_func('\t' + line)
+
+
+def generate_dist_folder_name(base_dist_name, arch_names=None):
+    """Generate the distribution folder name to use, based on a
+    combination of the input arguments.
+
+    Parameters
+    ----------
+    base_dist_name : str
+        The core distribution identifier string
+    arch_names : list of str
+        The architecture compile targets
+    """
+    if arch_names is None:
+        arch_names = ["no_arch_specified"]
+
+    return '{}__{}'.format(
```

Comment:
Dumb question: Why the double underscore?
Should we copy/paste that same function on buildozer or can you think a way of reusing it? e.g. maybe exposing it in the CLI so buildozer can use it?

Reply:
Double underscore is just my habit for separating things where each entry can plausibly contain underscores (as the dist name certainly can).

Reply:
OK that makes complete sense, thank you!